lib/dataBroker.js

```diff
@@ -56,7 +56,7 @@ module.exports = function(streamDAO) {
             if (!schema.isValidVersion(versionNum, uploadConfig.minimumUploaderVersion)) {
               return cb({
                 statusCode: 400,
-                text: 'The minimum supported version is ['+uploadConfig.minimumUploaderVersion+']. Version ['+datum.version+'] is no longer supported.',
+                message: 'The minimum supported version is ['+uploadConfig.minimumUploaderVersion+']. Version ['+datum.version+'] is no longer supported.',
```

Comment:
@jebeck and I discussed and decision was fixing the original issue of errors not being surface was the focus so I will leave this as updated.